src/Kdyby/Validator/DI/ValidatorExtension.php

```diff
@@ -82,6 +84,17 @@ public function loadConfiguration()
 			->setClass('Symfony\Component\Validator\Validator\ValidatorInterface')
 			->setFactory('Symfony\Component\Validator\Validator\RecursiveValidator');
 
+		Validators::assertField($config, 'strictEmail', 'boolean');
+
+		$builder->addDefinition($this->prefix('constraint.email'))
+			->setClass('Symfony\Component\Validator\Constraints\EmailValidator')
+			->setArguments(array(
```

Comment:
Yes but the compatibility from last stable release is already broken by the usage of a tag. This only breaks compatibility with previous master builds.